### ISSUE TITLE
Powerline warning in new tab

### ISSUE THREAD

**Issue by ayush000** [unixorn/zsh-quickstart-kit#103]
Hello,

I've been using this starter kit for a few days. I love it mainly because of async loading of plugins. Few problems I encountered:

- I see these logs: `Identity added: ...` listing down path to my ssh key every time I open a new terminal window.  I have added `export DONT_PRINT_SSH_KEY_LIST=true` in `~/.zshrc.d/my-conf.sh` but still see this log.
- Due to above print output, powerline gives a huge log on every new terminal window that looks like this:
```bash
[WARNING]: Console output during zsh initialization detected.
When using Powerlevel10k with instant prompt, console output during zsh
initialization may indicate issues.

You can:

  - Recommended: Change ~/.zshrc so that it does not perform console I/O
    after the instant prompt preamble. See the link below for details.

    * You will not see this error message again.
    * Zsh will start quickly and prompt will update smoothly.

  - Suppress this warning either by running p10k configure or by manually
    defining the following parameter:

      typeset -g POWERLEVEL9K_INSTANT_PROMPT=quiet

    * You will not see this error message again.
    * Zsh will start quickly but prompt will jump down after initialization.

  - Disable instant prompt either by running p10k configure or by manually
    defining the following parameter:

      typeset -g POWERLEVEL9K_INSTANT_PROMPT=off

    * You will not see this error message again.
    * Zsh will start slowly.

  - Do nothing.

    * You will see this error message every time you start zsh.
    * Zsh will start quickly but prompt will jump down after initialization.
```
I tried adding typeset statement in `~/.zshrc.d/my-conf.sh` but that doesn't work..

- Not sure about this but I always get this warning in startup of Visual Studio Code: `Resolving your shell environment is taking very long. Please review your shell configuration.`

I'm on OSX Catalina 10.15.7. My zsh version is `zsh 5.8 (x86_64-apple-darwin19.6.0)`

**Comment by brandonxoliver** [comment]
Same issue here on big sur

**Comment by mingaldrichgan** [comment]
If that is the only terminal output you see, it should be safe to add `typeset -g POWERLEVEL9K_INSTANT_PROMPT=quiet` to suppress that warning. (That's what I did, anyway.)

**Comment by unixorn** [comment]
I'm torn on adding `typeset -g POWERLEVEL9K_INSTANT_PROMPT=quiet` by default - it'll keep you from seeing errors printed in new plugins you add to the list, or in files loaded from `~/.zshrc.d`

On the other hand, I'd like it to be quieter.

Suggestions on how to treat this by default?

**Comment by mingaldrichgan** [comment]
I don't think that should be added by default, for the reason you mentioned. It's easy enough to add it to a file in `~/.zshrc.d`. Perhaps you could suggest this in `Readme.md`?

**Comment by ayush000** [comment]
JFYI I'm no longer using ZSH. Please feel free to close this ticket as and when it seems appropriate to you.
